backend/src/assets/404.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Sleep API</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta
+      name="description"
+      content="Optimize your Pokémon Sleep cooking with Sleep API's simulator. Explore the in-depth cooking tier list and find the best teams for your cooking strategy."
+    />
+    <meta property="og:title" content="Pokémon Sleep Cooking Simulator & Tier List | Sleep API" />
+    <meta
+      property="og:description"
+      content="Optimize your Pokémon Sleep cooking with Sleep API's simulator. Explore the in-depth cooking tier list and find the best teams for your cooking strategy."
+    />
+    <meta property="og:image" content="https://api.sleepapi.net/sleepapi-banner.png" />
+    <meta property="og:url" content="https://sleepapi.net" />
+    <link rel="icon" href="/cook.ico" type="image/x-icon" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css"
+      integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="/global.css" />
+    <style>
+      #bannerImage {
+        max-width: 90%;
+        height: auto;
+        margin-left: 5%;
+        margin-right: 5%;
+      }
+    </style>
+  </head>
+
+  <body>
+    <div class="container" style="margin-bottom: 50px">
+      <header class="d-flex justify-content-center mb-2 mt-2">
+        <a href="/">
+          <img
+            id="bannerImage"
+            src="sleepapi-banner.png"
+            alt="Sleep API Cooking Simulator Logo"
+            title="Pokémon Sleep Cooking Simulator"
+          />
+        </a>
+      </header>
+      <div class="card-container">
+        <div class="card" style="margin: auto">
+          <div class="card-body">
+            <div class="row justify-content-center mb-2">
+              <h1 style="text-align: center; color: #ddd">Error 404</h1>
```

Comment:
I know I specifically requested ddd, but I think it probably looks better with white in this case since its h1 and logo text color above is also white

Reply:
we can leave the "To front page" text color ddd, that looks good